config/prow/testgrid/testgrid.yaml

```diff
@@ -969,149 +969,245 @@ dashboards:
     base_options: "exclude-filter-by-regex=Overall$&group-by-directory=&expand-groups=&sort-by-name="
 - name: knative-0.10
   dashboard_tab:
-  - name: serving
+  - name: serving-continuous
     test_group_name: ci-knative-serving-0.10-continuous
     base_options: "sort-by-name="
     alert_options:
       alert_mail_to_addresses: "knative-productivity-dev@googlegroups.com"
     num_failures_to_alert: 3
-  - name: eventing
+  - name: serving-dot-release
+    test_group_name: ci-knative-serving-0.10-dot-release
+    base_options: "sort-by-name="
+    alert_options:
+      alert_mail_to_addresses: "knative-productivity-dev@googlegroups.com"
+    num_failures_to_alert: 3
```

Comment:
Could you change this number to `1`? Our current strategy is sending alert for every release errors and 3 continuous test errors

Reply:
I don't think it matters as it occurs very infrequently, would rather not make any further "optimization" in code

Reply:
I see, makes sense.